.gitignore

```diff
@@ -10,3 +10,5 @@ connect.files
 connect.includes
 *.tar.gz
 .DS_Store
+ext/boost_macos_1.62.0
+ext/opencv_macos_3.2.0
```

Comment:
How about a "*" instead of a specific version and maybe instead of "macos" as well -- there are linux users, you know.